_training/howto-new-module.md

```diff
@@ -63,13 +38,6 @@ What to not edit
 
 * If you improve the HSF style and other general parts of the page, please commit your changes to the HSF style repository and then pull your changes from there (this way, all other modules benefit from the changes as well)
 
-Additional steps
-----------------
-
-* Set the ``gh-pages`` branch as the default branch (``Settings`` > ``Branches`` > ``Default branch``)
```

Comment:
Is this possible with `cookiecutter`? Do we need to retain this instruction somewhere, or is it shown as a step in the `cookiecutter` run?

Reply:
Ah you're right, maybe it's best to get rid of this page completely and simply make the link point to the cookiecutter repository.

Reply:
Else we'd just end up duplicating information